app/routes.js

```diff
@@ -4,13 +4,14 @@ const { healthCheck } = require('./controllers/healthCheck');
 const validateBySchema = require('./middlewares/validations');
 const { signUpSchema, signInSchema } = require('./schemas/users');
 const uniqueEmail = require('./middlewares/uniqueEmail');
+const auth = require('./middlewares/auth');
 
 exports.init = app => {
   app.get('/health', healthCheck);
   app.get('/weets', weetsController.generateWeet);
 
   app.post('/users', [validateBySchema(signUpSchema), uniqueEmail], usersController.signUp);
   app.post('/users/sessions', validateBySchema(signInSchema), usersController.signIn);
-  // app.put('/endpoint/put/path', [], controller.methodPUT);
+  app.get('/users', auth, usersController.usersList);
```

Comment:
You should schema validation for this endpoint as well, that checks that the pagination input is correct